src/eppo_value.ts

```diff
@@ -32,38 +33,39 @@ export class EppoValue {
   }
 
   static generateEppoValue(
```

Comment:
Now may a good time to rename to `valueOf()` which I think is more standard

Reply:
Done